src/app.js

```diff
@@ -112,7 +112,32 @@ class Application {
         var showOpenDialogOptions = { 
             properties: [ 'openFile' ], 
             filters: [
-                { name: 'All Model Files',  extensions: [ 'onnx', 'pb', 'h5', 'hdf5', 'json', 'keras', 'mlmodel', 'caffemodel', 'model', 'meta', 'tflite', 'lite', 'pt', 'pth', 'pkl', 'joblib', 'pbtxt', 'prototxt' ] }
+                { 
+                    name: 'All Model Files',  
```

Comment:
Undo this change. File extensions should not be `xml` or `dot` as this will lead users to load generic files causing files.

Reply:
please read the general comment below

Reply:
formatting is back to the original.

@lutzroeder is the discussion resolved now?